.github/workflows/coverage.yml

```diff
@@ -25,10 +25,25 @@ jobs:
         id: buildx
         uses: docker/setup-buildx-action@v1
 
+      - name: Generate keys
+        run: make keys
+
       - name: Build docker container
         run: |
           cd ci && docker build -t test_suite:latest .
-          docker run -d --name test_suite -p 4200:4200 -p 4201:4201 -p 4202:4202 test_suite:latest
+          docker run -d --name test_suite -p 4200:4200 -p 4201:4201 -p 4202:4202 -p 4203:4203 test_suite:latest
+
+      - name: Generate Cargo.lock
+        run: cargo generate-lockfile
+
+      - uses: actions/cache@v2
+        with:
+          path: |
+            ~/.cargo/bin/
+            ~/.cargo/registry/index/
+            ~/.cargo/registry/cache/
+            ~/.cargo/git/db/
+          key: ${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.lock') }}
```

Comment:
we could also think about committing the lock file. I already saw that with many other crates.

Reply:
A missing lock file would explain why the cache wasn't working.

Reply:
Yep, didn't think about it before. I just merged #114. Should we try to remove the lock file generation?

Reply:
> Yep, didn't think about it before. I just merged #114. Should we try to remove the lock file generation?

Woops, yup, you're right.